### PR TITLE
kvm -> dnvm

### PR DESCRIPTION
**Pull request by anurse**
@Eilon wanted me to have a stealthy branch name :P (not really)

/cc @muratg @ChengTian @Eilon @NTaylorMullen @troydai

### CODE REVIEW

File: test/ps1/_Execute-Tests.ps1 [hunk 1]
```diff
@@ -28,18 +28,18 @@ if(!$RunningInNewPowershell) {
 # Set defaults
 if(!$PesterPath) { $PesterPath = Join-Path $PSScriptRoot ".pester" }
 if(!$TestsPath) { $TestsPath = Join-Path $PSScriptRoot "tests" }
-if(!$TargetPath) { $TargetPath = Convert-Path (Join-Path $PSScriptRoot "../../src/kvm.ps1") }
+if(!$TargetPath) { $TargetPath = Convert-Path (Join-Path $PSScriptRoot "../../src/dnvm.ps1") }
 if(!$TestWorkingDir) { $TestWorkingDir = Join-Path $PSScriptRoot "testwork" }
 if(!$TestAppsDir) { $TestAppsDir = Convert-Path (Join-Path $PSScriptRoot "../apps") }
 
 # Configure the Runtimes we're going to use in testing. The actual runtime doesn't matter since we're only testing
 # that kvm can find it, download it and unpack it successfully. We do run an app in the runtime to do that sanity
```
Comment (BrennanConroy) [inline test/ps1/_Execute-Tests.ps1]: kvm -> dnvm

File: test/ps1/_Execute-Tests.ps1 [hunk 2]
```diff
@@ -65,35 +65,35 @@ if($Debug) {
...
 
 # Set up the user/global install directories to be inside the test work area
 $UserPath = "$TestWorkingDir\$RuntimeFolderName"
-$env:KRE_HOME=$UserPath
-$env:KRE_USER_HOME=$UserPath
+$env:DNX_HOME=$UserPath
+$env:DNX_USER_HOME=$UserPath
 mkdir $UserPath | Out-Null
 
 # Helper function to run kvm and capture stuff.
```
Comment (BrennanConroy) [inline test/ps1/_Execute-Tests.ps1]: kvm -> dnvm

File: test/sh/tests/alias/Aliases can be removed.sh [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+$_DNVM_COMMAND_NAME alias test_alias_unalias "$_TEST_VERSION"
 
 # Unalias it
-$_KVM_COMMAND_NAME unalias test_alias_unalias
+$_DNVM_COMMAND_NAME unalias test_alias_unalias
 
 # Check the alias file
-[ ! -e "$KVM_USER_HOME/alias/test_alias_unalias.alias" ] || die "test alias was not removed"
\ No newline at end of file
+[ ! -e "$KVM_USER_HOME/alias/test_alias_unalias.alias" ] || die "test alias was not removed"
```
Comment (BrennanConroy) [inline test/sh/tests/alias/Aliases can be removed.sh]: DNVM_USER_HOME

File: src/dnvm.ps1 [hunk 4]
```diff
@@ -72,27 +72,27 @@ if($BuildVersion.StartsWith("{{")) {
...
 
 Set-Variable -Option Constant "AsciiArt" @"
-   __ ___   ____  ___
-  / //_/ | / /  |/  /
- / ,<  | |/ / /|_/ / 
-/_/|_| |___/_/  /_/  
+   ___  _  ___   ____  ___
+  / _ \/ |/ / | / /  |/  /
+ / // /    /| |/ / /|_/ / 
+/____/_/|_/ |___/_/  /_/
```
Comment (BrennanConroy) [inline src/dnvm.ps1]: :+1: :)

Comment (anurse) [inline src/dnvm.ps1]: It took me longer than I care to admit to find the same ASCII Art generator I used for the old one :)

File: src/dnvm.ps1 [hunk 5]
```diff
@@ -72,27 +72,27 @@ if($BuildVersion.StartsWith("{{")) {
 $FullVersion="$ProductVersion-$BuildVersion"
 
 Set-Variable -Option Constant "CommandName" ([IO.Path]::GetFileNameWithoutExtension($ScriptPath))
-Set-Variable -Option Constant "CommandFriendlyName" "K Runtime Version Manager"
-Set-Variable -Option Constant "DefaultUserDirectoryName" ".k"
-Set-Variable -Option Constant "OldUserDirectoryName" ".kre"
-Set-Variable -Option Constant "RuntimePackageName" "kre"
+Set-Variable -Option Constant "CommandFriendlyName" ".NET Version Manager"
+Set-Variable -Option Constant "DefaultUserDirectoryName" ".dnx"
+Set-Variable -Option Constant "OldUserDirectoryNames" @(".kre",".k")
```
Comment (troydai) [inline src/dnvm.ps1]: do we add space after comma in ps1?

Comment (davidfowl) [inline src/dnvm.ps1]: yes

File: test/apps/TestApp/Program.cs [hunk 6]
```diff
@@ -3,7 +3,7 @@
 using Microsoft.Framework.Runtime;
 using Microsoft.Framework.Runtime.Common.CommandLine;
 
-namespace HelloK {
+namespace TestApp {
```
Comment (troydai) [inline test/apps/TestApp/Program.cs]: why do we make this change now?

**Comment by troydai** [comment]
Besides the minor issues, overall looks good to me.

File: test/apps/TestApp/Program.cs (hunk 6 above)
Comment (anurse) [inline test/apps/TestApp/Program.cs]: While I'm here, just removing arbitrary Ks

File: src/dnvm.ps1 (hunk 5 above)
Comment (anurse) [inline src/dnvm.ps1]: :+1:

File: src/dnvm.ps1 (hunk 4 above)
Comment (muratg) [inline src/dnvm.ps1]: "D" character's loop is incomplete. :trollface:

**Comment by muratg** [comment]
:shipit:

I didn't look much into the test changes, as long as they pass, it should be OK. :)

File: src/dnvm.ps1 [hunk 7]
```diff
@@ -72,27 +72,27 @@ if($BuildVersion.StartsWith("{{")) {
...
+Set-Variable -Option Constant "CommandPrefix" "dnvm-"
 Set-Variable -Option Constant "DefaultArchitecture" "x86"
 Set-Variable -Option Constant "DefaultRuntime" "clr"
 Set-Variable -Option Constant "AliasExtension" ".txt"
 
 # These are intentionally using "%" syntax. The environment variables are expanded whenever the value is used.
-Set-Variable -Option Constant "OldUserHome" "%USERPROFILE%\.kre"
-Set-Variable -Option Constant "DefaultUserHome" "%USERPROFILE%\.k"
-Set-Variable -Option Constant "HomeEnvVar" "KRE_HOME"
+Set-Variable -Option Constant "OldUserHomes" @("%USERPROFILE%\.kre","%USERPROFILE%\.k")
```
Comment (troydai) [inline src/dnvm.ps1]: Here's another one, add space after comma.

**Comment by troydai** [comment]
:shipit:

**Pull request merged**